problem-17/src/app/component/comment-view/comment-view.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, DestroyRef, inject} from '@angular/core';
+import {FormBuilder, FormControl, FormGroup, ReactiveFormsModule, Validators} from "@angular/forms";
+import {NgIf} from "@angular/common";
+import {CommentService} from "../../service/comment.service";
+import {CommentModel} from "../../model/comment.model";
+import {takeUntilDestroyed} from "@angular/core/rxjs-interop";
+
+@Component({
+  selector: 'app-comment-view',
+  standalone: true,
+  imports: [ReactiveFormsModule, NgIf],
+  template:
+    `
+      <p *ngIf="!nameForm">Use form builder to create the form!</p>
+      <form  *ngIf="nameForm"  class="main-container">
+        <label for="Title" >Comment Title: </label>
+        <input id="Title" type="text">
+        <label for="Body">Comment Body: </label>
+        <input id="Body" type="text">
+        <button type="submit">Submit!</button>
+        <p *ngIf="formSubmitted">Comment Submitted</p>
+      </form>
+  `,
+  styleUrl: './comment-view.component.scss'
+})
+export class CommentViewComponent {
+
+  formSubmitted: boolean = false;
+  nameForm: FormGroup | undefined = undefined;
+
+  private commentService: CommentService = inject(CommentService);
+  private destroyRef: DestroyRef = inject(DestroyRef);
+
+
+  constructor(private formBuilder: FormBuilder) {
```

Comment:
use inject method

Reply:
Changed all questions to use inject for dependency injection